pic-sure-hpds-ui/src/main/webapp/picsureui/psamaui/user/addUser.js

```diff
@@ -96,12 +99,31 @@ define(["backbone", "handlebars", "user/connections", "picSure/userFunctions", "
 			this.model.unset("selectedUser");
 			$(".close").click();
 		},
-		render: function(){
+		checkUserRoleOnAdminRoleChecked: function (event) {
```

Comment:
```suggestion
		_verifyChecked: (input) => function (event) {
			let checkbox = $(input);
			if (event.target.checked) {
				checkbox.prop('checked', true);
			}
		},
		checkUserRoleOnAdminRoleChecked: _verifyChecked('input[name*="Admin"]'),
		checkAdminRoleOnAdminRoleChecked: _verifyChecked('input[name="PIC-SURE User"]')
```

You might be able to generalize this logic, similar to above.

Reply:
Per office hours conversation, I didn't see the inverse comparison, and the generalizaiton doesn't add much in this case. :) No need to worry about this.